doc/source/reference/random/extending.rst

```diff
@@ -81,7 +81,7 @@ directly from the ``_generator`` shared object, using the `BitGenerator.cffi` in
 
 New BitGenerators
 -----------------
-`~Generator` can be used with user-provided `~BitGenerator`\ s. The simplest
+`Generator` can be used with user-provided `BitGenerator`\ s. The simplest
```

Comment:
it's already a problem before you made your changes so no need to fix if you don't want to, but I'd also add cross-references for the mentions of BitGenerator above. It's a little weird that this reference down here has one but all the ones above don't.

Reply:
I added cross references throughout the entire document. Here's an image of the first couple paragraphs (old on left, new on right). 

![extending](https://github.com/numpy/numpy/assets/1594818/ce752a82-b9c9-4be2-af04-96a4a39fd1b1)

I think I got them all.